tests/rws_tests.py

```diff
@@ -276,6 +276,84 @@ def test_expected_rationales_case(self):
         loaded_sets = rws_check.load_sets()
         self.assertEqual(rws_check.error_list, [])  
 
+class TestCheckAssociatedCount(unittest.TestCase):
+    def test_within_limit(self):
+        json_dict = {
+            "sets":
+            [
+                {
+                    "primary": "https://primary.com",
+                    "associatedSites": ["https://associated1.com"],
+                    "rationaleBySite": {}
+                }
+            ]
+        }
+        rws_check = RwsCheck(rws_sites=json_dict,
+                      etlds=None,
+                       icanns=set())
+        loaded_sets = rws_check.load_sets()
+        rws_check.check_associated_count(loaded_sets)
+        self.assertEqual(rws_check.associated_warning, [])
```

Comment:
Where is `rws_check.associated_warning` defined? It looks like you meant this to be `rws_check.warning_texts` instead, here and below. If so, please investigate why the tests passed despite this typo?